### PR TITLE
Run koji-containerbuild tests in Travis

### PR DESCRIPTION
**Pull request by vrutkovs**
This PR is building a centos/fedora container and runs koji-containerbuild tests
and reports coverage data.

Note, that currently tests are py2 only, as there is no py3 koji version in repos 
yet.

In order to report results this repo should be added to Travis CI and Coveralls

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -19,11 +19,7 @@ def run(self):
         'koji_containerbuild',
         'koji_containerbuild.plugins',
     ],
-    install_requires=[
-        "koji",
-        "osbs",
-        #"distutils",
-    ],
+    install_requires=[],
```
Comment (lcarva) [inline setup.py]: Finally someone had the courage to remove these :)

**Comment by lcarva** [comment]
"ImportError: No module named koji"

File: koji-containerbuild.spec [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 Name:           koji-containerbuild
 Version:        0.7.4
-Release:        1%{?dist}
+Release:        2%{?dist}
```
Comment (lcarva) [inline koji-containerbuild.spec]: Why is this needed?

Comment (vrutkovs) [reply koji-containerbuild.spec]: Not sure if its required, as spec was changed, but no actual release was made. I'll keep this in case a new fix release will be done

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # ## Linux
-.*
+.cache
+.coverage
+build/
 !.gitignore
```
Comment (rcerven) [inline .gitignore]: remove this as well, now without '.*' it isn't useful

Comment (vrutkovs) [reply .gitignore]: Fixed

File: .gitignore [hunk 4]
```diff
@@ -1,5 +1,7 @@
...
 /dist/
```
Comment (rcerven) [inline .gitignore]: this should be probably 'dist/'

Comment (lcarva) [reply .gitignore]: Slash in front of each item means only ignore starting at root of repo. It should probably stays with leading slash, and added to the other items too :)

File: koji-containerbuild.spec (hunk 2 above)
Comment (rcerven) [reply koji-containerbuild.spec]: this is always 1 in atomic-reactor and osbs-client, do we really wanna change this?

**Comment by lcarva** [comment]
Updated release notes draft:
> - Invalid dependencies removed from `setup.py`
> - Added `test/requirement.txt`

Not mentioning travis integration because that's irrelevant to released code.
